trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
```

Comment:
- [ ] Please change the above code with this to work the trimmer pattern to work as expected. `@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`.